react-delegationdashboard/src/components/Layout/APRCalculation.ts

```diff
@@ -55,8 +55,10 @@ const calculateAPR = ({
   const validatorBaseStake = allActiveNodes * stakePerNode;
   const validatorTotalStake = parseInt(denominateValue(totalActiveStake));
   const validatorTopUpStake = validatorTotalStake - allNodes * stakePerNode;
-
-  const validatorTopUpReward = (validatorTopUpStake / networkTopUpStake) * topUpReward;
+  let validatorTopUpReward = validatorTopUpStake / networkTopUpStake;
```

Comment:
The issue observed was that `networkTopUpStake` was 0, so division by 0 produced a NaN. If `topUpReward` was 0 then apr should have been displayed as 0.
I would suggest something like this:
`const validatorTopUpReward = networkTopUpStake > 0 ? (validatorTopUpStake / networkTopUpStake) * topUpReward : 0;`

However, the cause of the problem seems to have been that `networkTotalStake` was equal to `networkBaseStake`, which I find quite impossible... I think it needs to be investigated where those values are produced (in the dapp) @LucianMincu .